todo_app/templates/add.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <form action="" method="post">
+        <label for="titleInput">titleInput</label>
+        <input type="text" id="titleInput" name="title">
```

Comment:
Really minor, but adding a `required` keyword to this can give some simple browser-side protection against users accidentally submitting an empty input. This doesn't stop determined users who can avoid it, so it's not a security approach, but it can make for much nicer user experience